Practice Problems/LinkedList/README.md

```diff
@@ -0,0 +1,6 @@
+# LinkedList
+All questions added here are taken from various sites like GeeksForGeeks, LeetCode, Hackerrank, CodeForces, etc.
+
+## Problems
+- [Hamming Distance](https://github.com/srsandy/Data-Structures-and-Algorithms-in-Java-2nd-Edition-by-Robert-Lafore/tree/master/Practice%20Problems/LinkedList/Reverse%20LinkedList)
```

Comment:
Can you pls rename this to Reverse LinkedList and not Hamming Distance

Reply:
> Can you pls rename this to Reverse LinkedList and not Hamming Distance

done